### PR TITLE
Gdorsey

### PR DESCRIPTION
**Pull request by gregorydorseyrvature**

### CODE REVIEW

File: ProjectForce/force-app/main/default/aura/modal/modalController.js [hunk 1]
```diff
@@ -56,12 +56,15 @@
     },
 
     showModal : function(component, event, helper) {
+        console.log("showModal");
```
Comment (jiphan) [inline ProjectForce/force-app/main/default/aura/modal/modalController.js]: please remove console.logs

File: ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js [hunk 2]
```diff
@@ -226,13 +250,14 @@ myAction : function(component, event, helper) {
...
             enabled: false
+
     }        
 };
     
     //calls the apex controller and runs the method getTrainings then saves the return of the method into getTracks
     var getTracks = component.get("c.getTrainings");
     //run a callback
-    getTracks.setCallback(this, function(response) {
+    getTrrrack.setCallback(this, function(response) {
```
Comment (jiphan) [inline ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js]: old code, please merge production into your branch

File: ProjectForce/force-app/main/default/aura/modal/modalHelper.js [hunk 3]
```diff
@@ -79,22 +84,93 @@
...
 		
+		console.log("locationList:"+locationList);
 
 		if(!locationList){
 			component.set("v.locations", location);
 		}
 		component.set("v.selectedLocation", location[0].Name);
-		
+
+		/*if(state === "SUCCESS"){
```
Comment (mruizrevature) [inline ProjectForce/force-app/main/default/aura/modal/modalHelper.js]: Please remove commented code

File: ProjectForce/force-app/main/default/aura/newD3Comp/newD3Comp.css [hunk 4]
```diff
@@ -67,7 +67,6 @@
     border-radius: 5px;
 	background-color: #F26925;
```
Comment (mruizrevature) [inline ProjectForce/force-app/main/default/aura/newD3Comp/newD3Comp.css]: Don't include this file. It's not detrimental to the feature.

File: ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js [hunk 5]
```diff
@@ -196,27 +207,40 @@ myAction : function(component, event, helper) {
...
                     ticks: {
                         fontFamily: "'Futura', sans-serif",
-                        fontSize:11,
-                        display: false
-                        
-                        
+                        fontSize:20,
+                        display: true,
+                        callback: function(value, index, values) {
+                            var tra = JSON.parse(localStorage.getItem("tracks"));// getting values from helper
```
Comment (mruizrevature) [inline ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js:215]: Call it something other than tra

Comment (mruizrevature) [inline ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js:215]: Please put inline comments above the thing it's talking about. It's the convention followed elsewhere

File: ProjectForce/force-app/main/default/classes/GantController.cls [hunk 6]
```diff
@@ -9,7 +9,8 @@ public with sharing class GantController {
             'Project__c',
             'Room__c',
             'Trainer__c',
-            'projectstartdate__c'
+            'projectstartdate__c',
+            'Track__c' //added
```
Comment (mruizrevature) [inline ProjectForce/force-app/main/default/classes/GantController.cls]: Don't need a comment that says added

File: ProjectForce/force-app/main/default/profiles/profiles/Admin.profile-meta.xml [hunk 7]
```diff
@@ -283,6 +283,10 @@
         <enabled>true</enabled>
         <name>CreateLtngTempFolder</name>
     </userPermissions>
+    <userPermissions>
```
Comment (mruizrevature) [inline ProjectForce/force-app/main/default/profiles/profiles/Admin.profile-meta.xml]: Are these permissions necessary? What are they used for?

**Review by mruizrevature — CHANGES_REQUESTED** [review]
Please change the name of this Pull Request to reflect what it is that you are requesting be added. Likewise, you have a lot of useless console.log statements and commented out code.

**Pull request closed without merging**